build/Dockerfile

```diff
@@ -46,7 +46,10 @@ ENV CI ${CI}
 ADD . /workspace/kyuubi
 WORKDIR /workspace/kyuubi
 
-RUN ./build/dist ${MVN_ARG} && \
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive \
+    apt-get install -y python3 && \
+    ./build/dist ${MVN_ARG} && \
```

Comment:
why we do the change ?

Reply:
Add a new `build/release/append_notice.py` to reduce the redundant NOTICE